src/app/(tabs)/search/index.tsx

```diff
@@ -77,7 +74,12 @@ function SearchScreen() {
             image={item.featured_media}
             authorImage={item.author_image}
             tags={item.genre_medium}
-            pressFunction={() => null}
+            pressFunction={() =>
+              router.push({
```

Comment:
onPress function works on my end for each of the story cards and the story pages display the correct info!